src/app/screens/Home/components/BookList/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// eslint-disable-next-line import/namespace
+import { useRequest } from '~app/hooks/useRequest';
+import { getBooks } from '~services/BooksService';
+import Loading from '~components/Spinner/components/loading';
+
+import styles from './styles.module.scss';
+import Book from './components/Book';
+
+export default function BookList() {
+  const [state, loading] = useRequest(
+    {
+      request: getBooks,
+      payload: []
+    },
+    []
+  );
+
+  return (
+    <div className={styles.bookList}>{loading ? <Loading /> : state && <Book books={state.page} />}</div>
```

Comment:
why dont use the map here and call `<Book` its wrong to name a component with singular name if you map its list inside the component